docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -232,15 +229,15 @@ yarn add @material-ui/styles@next
 
 ### ExpansionPanel
 
-- [ExpansionPanelActions] Rename the `action` CSS class `spacing`.
+- [ExpansionPanelActions] Rename the `action` CSS class to `spacing`.
 - [ExpansionPanel] Increase the CSS specificity of the `disabled` style rule.
 
 ### List
 
 - [List] Rework the list components to match the specification:
 
-  - The usage of the `ListItemAvatar` component is required when using an avatar
-  - The usage of the `ListItemIcon` component is required when using a left checkbox
+  - The usage of the `ListItemAvatar` component is required when using an avatar`.
+  - The usage of the `ListItemIcon` component is required when using a left checkbox.
```

Comment:
```suggestion
  - The use of the `ListItemIcon` component is required when using a left checkbox.
```

@oliviertassinari Again? 😉

Reply:
I never known that there's a difference between two until now  😂